src/plugins/console/public/application/containers/editor/monaco_editor_output_actions_provider.ts

```diff
@@ -30,30 +30,45 @@ export class MonacoEditorOutputActionsProvider {
     private setEditorActionsCss: (css: CSSProperties) => void
   ) {
     this.highlightedLines = this.editor.createDecorationsCollection();
-    this.editor.focus();
```

Comment:
Big part of the problem was that when we toggle between `empty state`, `loading state` and `editor` the constructor of the editor always kicks in and focuses the output panel highlighting whatever response was sent. I've now removed this and made it so that decorations only take effect when the editor is manually focused.